yaml/landuse.yaml

```diff
@@ -489,6 +493,17 @@ filters:
       <<: *output_properties
       kind: glacier
       tier: 3
+  # reef
+  - filter: {natural: reef}
```

Comment:
Odd question – but since this is underwater should this go into the **water** layer instead of **landuse**?

Reply:
Sure, that makes sense. Moved to `water` layer in 973963c.